dao/src/integ-test/resources/uk/gov/moj/sdt/dao/sql/PurgeDaoTest.sql

```diff
@@ -60,6 +71,9 @@ INSERT INTO individual_requests(
 INSERT INTO individual_requests(
   individual_request_id, bulk_submission_id, customer_request_ref, request_status, sdt_bulk_reference, line_number, sdt_request_reference, created_date, updated_date, completed_date, forwarding_attempts, dead_letter, internal_system_error, request_type, version_number, individual_payload, target_application_response)
   VALUES (111, 17, 'CR104', 'COLD', 'BR105', 1, 'RR204', now(), now(), now(), 3, 'N', 'Error this is indeed', 'ReqType1', 1, 'IndPL_1', 'TAppResp6');
+INSERT INTO individual_requests(
+individual_request_id, bulk_submission_id, customer_request_ref, request_status, sdt_bulk_reference, line_number, sdt_request_reference, created_date, updated_date, completed_date, forwarding_attempts, dead_letter, internal_system_error, request_type, version_number, individual_payload, target_application_response)
+VALUES (112, 13, 'CR104', 'COLD', 'BR105', 1, 'RR204', now(), now(), now(), 3, 'N', 'Error this is indeed', 'ReqType1', 1, 'IndPL_1', 'TAppResp6');
```

Comment:
changed :

INSERT INTO error_logs(
  error_log_id, individual_request_id, error_code, created_date, updated_date, version_number, error_text)
  VALUES (1006, 109, 'CODE4', (now() - INTERVAL '20 day'), now(), 1, 'Error number 3');
  
  To 
  
  INSERT INTO error_logs(
  error_log_id, individual_request_id, error_code, created_date, updated_date, version_number, error_text)
  VALUES (1006, 112, 'CODE4', (now() - INTERVAL '20 day'), now(), 1, 'Error number 3');
  
  
  you know this is issue you are getting when you run the Drurun (because of the association we have added as part of JPA migration)